frontend/packages/console-telemetry-plugin/src/listeners/segment.ts

```diff
@@ -110,11 +110,11 @@ export const eventListener: TelemetryEventListener = async (
   eventType: string,
   properties?: any,
 ) => {
-  if (TELEMETRY_DISABLED || !apiKey) {
+  if (!apiKey) {
     if (TELEMETRY_DEBUG) {
       // eslint-disable-next-line no-console
       console.debug(
-        'console-telemetry-plugin: telemetry disabled - ignoring telemetry event:',
+        'console-telemetry-plugin: missing Segment API key - ignoring telemetry event:',
```

Comment:
missing i18n `.json` changes

Reply:
this change doesn't include any translated strings. We don't usually translate browser logs.